packages/driver/src/cypress/error_messages.ts

```diff
@@ -2315,22 +2315,38 @@ export default {
         \`url\` from the \`url\` parameter: {{url}}`,
       docsUrl: 'https://on.cypress.io/visit',
     },
-    loading_network_failed: stripIndent`\
-      ${cmd('visit')} failed trying to load:
+    loading_network_failed (obj) {
+      const TLS_VERSION_ERROR_RE = /TLSV1_ALERT_PROTOCOL_VERSION|UNSUPPORTED_PROTOCOL|TLSV1_ALERT_INTERNAL_ERROR/
 
-      {{url}}
+      const isTlsVersionError = TLS_VERSION_ERROR_RE.test(obj.error.message)
 
-      We attempted to make an http request to this URL but the request failed without a response.
+      // if we detect there are TLS v1/v1.1 errors, we want to direct users on
+      // ways they might be able to resolve their issue as the default error is cryptic
+      // TODO: UPDATE system-tests/test/visit_spec.js TLSv1 test to check for this error once on Electron 25 / Node 18
```

Comment:
ill update the system test in the electron upgrade that actually checks for the printing of this message

Reply:
Actually, this has nothing to do with the shipped cypress node version. As lonmg as the users origin server is running a node version where they can support tls 1 or 1.1 this will be fine.